src/rust/include/rust/VA_OPT.hpp

```diff
@@ -0,0 +1,152 @@
+//  Copyright (c) 2019 Will Wray https://keybase.io/willwray
+//
+//  Distributed under the Boost Software License, Version 1.0.
+//        http://www.boost.org/LICENSE_1_0.txt
+//
+//  Repo: https://github.com/willwray/VA_OPT
+
+#pragma once
```

Comment:
This is the only use of `#pragma once` in our own code, and it is not standard C++20. (It's also used in Boost.)
```suggestion
#ifndef ZCASH_RUST_INCLUDE_RUST_VA_OPT_H
#define ZCASH_RUST_INCLUDE_RUST_VA_OPT_H
```